### PR TITLE
feat: add mergeProps

### PR DESCRIPTION
**Pull request by char0n**
Ref https://github.com/ramda/ramda/issues/2295

### CODE REVIEW

**Comment by char0n** [comment]
@BjornMelgaard tommorow I will also add [mergePaths](https://github.com/char0n/ramda-adjunct/issues/136) and we can go for a v1.17.0 release so that you can use `mergeProps` and `mergePaths` right away. If you want you can provide PR your self, but let me know in advance.

Good job and thank you for contributing.

**Comment by BjornMelgaard** [comment]
Ok, I'll add mergePaths

**Comment by char0n** [comment]
@BjornMelgaard can we merge this one ?

**Comment by BjornMelgaard** [comment]
yea, sure, you can do whatever you want)

File: src/mergeProps.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+/**
+ * Functional equivalent of merging object properties with object spread operator.
+ *
+ * @func mergeProps
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/1.17.0|v1.17.0}
+ * @category Object
+ * @sig [k] -> {k: {a}} -> {a}
+ * @see {@link RA.mergePaths|mergePaths}
```
Comment (BjornMelgaard) [inline src/mergeProps.js]: @char0n  mergePaths????

Comment (char0n) [reply src/mergeProps.js]: Yup, mergeProps should link to mergePaths and vice versa. Utils should link to related utils, not to their implementation details. Does it make sense ?

**Pull request merged**